matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/profile/DefaultProfileService.kt

```diff
@@ -73,4 +80,31 @@ internal class DefaultProfileService @Inject constructor(private val taskExecuto
                 }
                 .executeBy(taskExecutor)
     }
+
+    override fun getThreePids(): List<ThreePid> {
+        return monarchy.fetchAllMappedSync(
+                { it.where<UserThreePidEntity>() },
+                { it.asDomain() }
+        )
+    }
+
+    override fun getThreePidsLive(): LiveData<List<ThreePid>> {
+        // Force a refresh of the values
+        refreshUserThreePidsTask
```

Comment:
i'd prefer a boolean to trigger or not the refresh;

Reply:
Added

Reply:
(but I think the value should always be true, because, those data does not comes from the sync)